### DISCUSSION TITLE
Why is /player/ranked/skywars a cached endpoint?

### DISCUSSION THREAD

**Discussion by justinrest** [HypixelDev/PublicAPI#505] [General]
/player/ranked/skywars should not be cached at all considering it uses one API key use,

https://hypixel.net's Leaderboard for rating is not cached nor is /stats on Hypixel.

![image](https://user-images.githubusercontent.com/92120745/149645561-8d8858ca-bb52-4f87-9980-29cbe28e2748.png)

![image](https://user-images.githubusercontent.com/92120745/149645564-5ba4d3e1-22a1-47b4-929a-f76a2f27a495.png)

**Comment by mdashlw** [comment]
I agree there should not be any caching for this endpoint, however see https://github.com/HypixelDev/PublicAPI/discussions/498

**Reply by justinrest** [reply]
> I agree there should not be any caching for this endpoint, however see #498

Oh I didn't see this, that is sad that this endpoint will remain cached.
